docs/source/build_a_lint_rule.ipynb

```diff
@@ -211,7 +183,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
```

Comment:
`execution_count` is assigned at run time, the value should be `null`.

Reply:
When any of the cell is already executed, NBSphinx won't run the notebook in CI.
![image](https://user-images.githubusercontent.com/3840867/92536059-91918e00-f1ed-11ea-8089-13e7ed35bb66.png)

Reply:
Okay, will make sure, NBSphinx new to me, that's why. Thank you for helping me with this.

Reply:
done